code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -9,9 +9,6 @@
 	var/obj/item/handcuffed = null //Whether or not the mob is handcuffed
 	var/obj/item/legcuffed = null  //Same as handcuffs but for legs. Bear traps use this.
 
-	var/list/features = list("mcolor" = "FFF", "tail" = "Smooth", "snout" = "Round", "horns" = "None", "frills" = "None", "spines" = "None", "body_markings" = "None")
-
-

```

Comment:
The only reason features is also carbon defined is because we can't call up a character's DNA in the preference screen before the mob exists. If you fixed/can fix that then yeah it's more than welcome to go.